api/src/services/listing-csv-export.service.ts

```diff
@@ -877,6 +877,11 @@ export class ListingCsvExporterService implements CsvExporterServiceInterface {
           format: (val: User[]): string =>
             val.map((user) => `${user.firstName} ${user.lastName}`).join(', '),
         },
+        {
+          path: 'isVerified',
```

Comment:
Issue: This column should only appear if the `enableIsVerified` is enabled. You can reference how the `homeType` column is working above on how to conditionally show this